lib/libvcc/vcc_vmod.c

```diff
@@ -106,6 +99,21 @@ vcc_ParseImport(struct vcc *tl)
 	}
 
 	SkipToken(tl, ';');
+	if (osym != NULL) {
+		AN(osym->path);
+		if (strcmp(osym->path, fn) != 0) {
+			VSB_printf(tl->sb, "Warning: Module %.*s imported from different location.\n",
+			    PF(mod));
+			vcc_ErrWhere2(tl, t1, tl->t);
+			VSB_printf(tl->sb, "Previous import was here:\n");
+			vcc_ErrWhere2(tl, osym->def_b, osym->def_e);
+			tl->err = 0; // reset error; this is just a warning
```

Comment:
]] Kacper Why 


> There is a use case; let's say you ship a vmod zoo and vcl zaa, and you want to test it without installing the vmod; then you either have
> to modify the vcl to say "import zoo from bar" or you can, in your vtc specify
> 
>    import zoo from bar; include "zaa.vcl";

A better solution then is to run Varnish with a different search path
for vmods when testing.

-- 
Tollef Fog Heen
UNIX is user friendly, it's just picky about who its friends are